internal/cli/atlas/setup/setup_cmd.go

```diff
@@ -159,7 +159,6 @@ func Builder() *cobra.Command {
 	// Register and login related
 	cmd.Flags().BoolVar(&loginOpts.IsGov, "gov", false, "Register to Atlas for Government.")
 	cmd.Flags().BoolVar(&loginOpts.NoBrowser, "noBrowser", false, "Don't try to open a browser session.")
-	cmd.Flags().BoolVar(&loginOpts.SkipConfig, "skipConfig", false, "Skip profile configuration.")
```

Comment:
We need to follow our guidelines for deprecating the flag - sending the wiki link in private

Reply:
+1

Reply:
done, thx